docs/contributing/development-policies.rst

```diff
@@ -94,18 +94,12 @@ Branches
 
 We maintain a number of branches on `our GitHub repository <https://github.com/divio/django-cms>`_.
 
-the latest (highest-numbered) ``release/x.y.z``
-    This is the branch that will become the next release on PyPI.
-
-    **Fixes and backwards-compatible improvements** (i.e. most pull requests) will be made against
-    this branch.
-
 ``develop``
-    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
-    ``release/x.y.z``.
+    The default target branch for on-going development.
 
-    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
-    substantial team co-ordination.
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI. ``develop`` will be merged into ``release/x.y.z``
```

Comment:
`develop` is/should never be merged into `release/x.y.z`, instead commits are cherry-picked